src/snipe/cli/cli_ops.py

```
@@ -0,0 +1,869 @@
+    # Define a mapping of option names to operation identifiers
+    option_order = {
+        '--reset-abundance': ('reset_abundance', None),
+        '--trim-singletons': ('trim_singletons', None),
+        '--min-abund': ('keep_min_abundance', 'min_abund'),
+        '--max-abund': ('keep_max_abundance', 'max_abund'),
+        '--trim-below-median': ('trim_below_median', None),
+    }
+
+    # Iterate through argv to capture the order of operations
+    skip_next = False
+    for i, arg in enumerate(argv):
```

Ruff

Loop control variable `i` not used within loop body

(B007)